jfrog-cli/artifactory/commands/docker/push.go

```diff
@@ -16,11 +19,19 @@ type DockerPushConfig struct {
 
 // Push docker image and create build info if needed
 func PushDockerImage(imageTag, targetRepo, buildName, buildNumber string, config *DockerPushConfig) error {
+	// Perform login
+	loginConfig := &dockerLoginConfig{ArtifactoryDetails: config.ArtifactoryDetails}
+	err := dockerLogin(imageTag, loginConfig)
+	if err != nil {
+		cliutils.ExitOnErr(err)
```

Comment:
You should return the error here and handle it in cli.go.

Reply:
fixed